src/main/java/com/rev_connect_api/dto/PostCreateRequest.java

```diff
@@ -0,0 +1,32 @@
+package com.rev_connect_api.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+public class PostCreateRequest {
+
+    @NotEmpty(message = "title is blank")
+    private String title;
+    @NotEmpty(message = "content is blank")
+    private String content;
+
+    public PostCreateRequest() {}
```

Comment:
You can use the `@Data` annotation to avoid constructors and getters/setters :) 
`lombok.Data
Generates getters for all fields, a useful toString method, and hashCode and equals implementations that check all non-transient fields. Will also generate setters for all non-final fields, as well as a constructor (except that no constructor will be generated if any explicitly written constructors already exist).`